index.html

```diff
@@ -155,6 +148,259 @@ <h2 class="title text-center">About DjangoCongress JP<br /><small>DjangoCongress
+                    <!--<a href="https://cybozu.co.jp/company/access/tokyo/" target="_blank">Access Map</a>-->
+          		    </p></dd>
+                  <dt><i class="fa fa-fw fa-building"></i>&nbsp;参加人数 / Participants</dt>
+                  <dd><p>TBD (about 100) people</p></dd>
+                  <dt><i class="fa fa-fw fa-tag"></i>公式ハッシュタグ / Twitter Hash Tag</dt>
+                  <dd><p><a target="_blank" href="https://twitter.com/search?f=tweets&amp;vertical=default&amp;q=djangocongress&amp;src=typd">#djangocongress</a></p></dd>
+                </dl>
+              </div>
+              <div class="col-md-6 align-self-center">
+                <iframe src="https://www.google.com/maps/embed?pb=!1m14!1m8!1m3!1d12804.052478359754!2d138.187394!3d36.650135!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x0%3A0x614f4a331ea68076!2z6ZW36YeO5biCIOeUn-a2r-Wtpue_kuOCu-ODs-OCv-ODvA!5e0!3m2!1sja!2sjp!4v1576325692747!5m2!1sja!2sjp" width="100%" height="450" frameborder="0" style="border:0;" allowfullscreen></iframe>
+                <p><a href="https://goo.gl/maps/XD7md2hBbY2F3grs8" target="_blank">Open in Google Map</a></p>
+              </div>
+            </div>
+          </div>
+        </div><!-- end .section-about -->
+
+        <!-- section-program -->
+        <div id="program" class="section section-program">
+          <div class="container">
+            <div class="tim-title">
+              <h2 class="title text-center">PROGRAM<br /><small>プログラム</small></h2>
+            </div>
+
+            <h2 class="title text-center"><small>Coming Soon</small></h2>
+
+            <!-- <p class="text-center">
+              プロポーザル募集 開始時期: 2月上旬~中旬予定
+            </p>
+            <p class="text-center">
+              testsss
+            </p> -->
+
+            <div class="title text-center">
+              <P class="text-center">
+                プロポーザル募集 開始時期
+              </P>
+              <P class="text-center">
+                3月 上旬~中旬
+              </P>
+              <br/>
+
+              <P class="text-center">
+                Proposal Recruitment
+              </P>
+              <P class="text-center">
+                early March ~ mid March
+              </P>
+            </div>
+          </div>
+        </div><!-- end .section-program -->
+
+        <!-- section-ticket -->
+        <div class="section section-info" id="tickets">
+          <div class="container">
+            <div class="row">
+              <div class="col-md-9 ml-auto mr-auto ">
+                <h2 class="title text-center">GET YOUR TICKET<br /><small>チケット</small></h2>
+                <p>
+                  参加するにはチケットが必要です。<br />発表者と海外からの参加者は、参加費が無料になります。
+                </p>
+                <p>
+                  Tickets are now on sale! Buy yours today! <br />
+                  The selected speakers and attendees from outside of tokyo/international will gain free entrance to the conference.
```

Comment:
The selected speakers and attendees from outside of tokyo/international
↓
The selected speakers and attendees from outside of Japan